### ISSUE TITLE
Screen readers can't infer names from various "icon-only" controls

### ISSUE THREAD

**Issue by zersiax** [caido/caido#1076]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Bug description

Various elements within the Caido UI currently have no accessible name, making it impossible for various assistive technologies to figureo ut what a button/control is going to do when clicked/activated

### How to reproduce

WIth a screen reader running (narrator/NVDA on Windows, Orca on GNome/KDE, VOiceOver on Mac) navigate across the various social media icons on the Caido welcome screen.
No names/descriptions for these will be provided.

### Environment & setup

Irrelevant, this is a web standards issue.
### Remediation

The easiest way to fix this is to provide labels of some kind for these. This can be a visual label, so everyone can see what a logo/icon means without needing a particular context, or by using the aria-label HTML attribute which gives a screenreader-accessible override to the element in question, e.g. 
```html
<i class="super-fancy-twitter-icon" aria-label="Follow us on X!"></i>
```

**Comment by zersiax** [comment]
I'm curious why this issue has been designated as high effort? In most cases all the offending elements need is a single extra HTML attribute, feels like a good first issue to me?

**Comment by Sytten** [comment]
Its just that there are a lot of components to touch before we can consider the issue done. But it will rollout slowly with each release once we get going :)
